### ISSUE TITLE
Proxy not working since v5.0.0

### ISSUE THREAD

**Issue by mario-moura-silva** [auth0/auth0-deploy-cli#260]
### Description

> I'm not able to use the proxy parameter since v5.0.0

### Reproduction

```
Executing "step_script" stage of the job script
 $ a0deploy export -c config.json -d -f yaml -o dump -p $HTTP_PROXY
 2020-07-14T08:40:19.729Z - info: Creating dump
 2020-07-14T08:42:28.951Z - error: Problem running command export
 2020-07-14T08:42:28.952Z - error: connect ETIMEDOUT **.**.**.**:***
 ERROR: Job failed: exit code 1
```

**Comment by fromeroy** [comment]
Hi @mario-moura-silva , this issue has been fixed within the last release v5.1.5

**Comment by mario-moura-silva** [comment]
Thank you @fromeroy
